### PR TITLE
tests: adding spread support for ubuntu mantic

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Spread support for ubuntu mantic

### CODE REVIEW

**Review by mvo5 — APPROVED** [review]
Thank you!

File: spread.yaml [hunk 1]
```diff
@@ -143,6 +143,9 @@ backends:
             - ubuntu-23.04-64:
                   storage: 12G
                   workers: 8
+            - ubuntu-23.10-64:
+                  storage: 12G
+                  workers: 8
```
Comment (mvo5) [inline spread.yaml:148]: Let's please also add it to the qemu backends

Comment (sergiocazzolato) [reply spread.yaml:148]: done, thanks for the review

**Review by MiguelPires — APPROVED** [review]
LGTM

**Review by Meulengracht — APPROVED** [review]
LGTM

**Pull request merged**